python/dask_cudf/dask_cudf/io/tests/test_json.py

```diff
@@ -71,3 +71,22 @@ def test_read_json_lines(lines):
         actual = dask_cudf.read_json(f, orient="records", lines=lines)
         actual_pd = pd.read_json(f, orient="records", lines=lines)
         dd.assert_eq(actual, actual_pd)
+
+
+@pytest.mark.filterwarnings("ignore:Using CPU")
+def test_read_json_nested_experimental():
+    # Check that `engine="cudf_experimental"` can
+    # be used to support nested data
+    df = pd.DataFrame(
+        {
+            "a": [{"y": 2}, {"y": 4}, {"y": 6}, {"y": 8}],
+            "b": [[1, 2, 3], [4, 5], [6], [7]],
+            "c": [1, 3, 5, 7],
+        }
+    )
+    lines = dict(orient="records", lines=True)
+    with tmpfile("json") as f:
```

Comment:
Use the `tmp_path` fixture to get a temporary `pathlib.Path` object that pytest manages and will clean up afterwards (but leaves lying around in the case that the test fails, which can be useful):

```python
def test_read_json_nested_experimental(tmp_path):
    ...
    with tmp_path / "data.json" as f:
         ...
```

Reply:
Ah - lazy copy-pasting on my end. Thanks for catching this!